workitems/tokenlist/l2-token-list-v1.0-psd01.md

```diff
@@ -0,0 +1,723 @@
+                        "decimals",
+                        "createdAt",
+                        "updatedAt"
+                    ],
+                    "properties": {
+                        "chainId": {
+                            "$comment": "{\"term\": \"chainId\", \"@id\": \"https://schema.org/identifier\"}",
+                            "title": "chainId",
+                            "description": "The typically used number identifier for the chain on which the token was issued.",
+                            "type": "number",
+                            "examples": [137]
+                        },
+                        "chainURI": {
+                            "$comment": "{\"term\": \"chainURI\", \"@id\": \"https://schema.org/identifier\"}",
+                            "title": "chainURI",
+                            "description": "A resolvable URI to the genesis block of the chain on which the token was issued following the RFC 3986 standard.",
+                            "type": "string"
+                             "examples": ["https://polygonscan.com/block/0"]
+                        },
+                        "genesisBlockHash": {
+                            "$comment": "{\"term\": \"genesisBlockHash\", \"@id\": \"https://schema.org/sha256\"}",
+                            "title": "genesisBlockHash",
+                            "description": "The hash of the genesis block of the chain on which the token was issued.",
+                            "type": "string",
+                            "examples": ["0xa9c28ce2141b56c474f1dc504bee9b01eb1bd7d1a507580d5519d4437a97de1b"]
+                        },
+                        "tokenIssuerId": {
+                            "$comment": "{\"term\": \"tokenIssuerId\", \"@id\": \"https://schema.org/identifier\"}",
+                            "title": "tokenIssuerId",
+                            "description": "A resolvable URI identifying the token issuer following the RFC 3986 standard.",
+                            "type": "string",
+                            "examples": ["https://polygonscan.com/address/0xa9c28ce2141b56c474f1dc504bee9b01eb1bd7d1a507580d5519d4437a97de1b"]
+                        },
+                        "tokenIssuerName": {
+                            "$comment": "{\"term\": \"tokenIssuerName\", \"@id\": \"https://schema.org/name\"}",
+                            "title": "tokenIssuerName",
+                            "description": "The name oof the token issuer.",
+                            "type": "string"
+                            "examples": ["Matic"]
+                        },
+                        "tokenId": {
+                            "$comment": "{\"term\": \"tokenId\", \"@id\": \"https://schema.org/identifier\"}",
+                            "title": "tokenId",
+                            "description": "A resolvable URI to a CanonicalTokenList entry of the token following the RFC 3986 standard.",
```

Comment:
I understand the other external resources, but what would this be pointing to in practice? Is there a single global CanonicalTokenList to reference?

Reply:
reworded to be not confusing